### DISCUSSION TITLE
impossible to select and kill processes

### DISCUSSION THREAD

**Discussion by patrcoil** [nicolargo/glances#1810] [Q&A]
Hi, Glances is amazing! unfortunately, unlikely to what is mentioned in all documentations I found, impossible to select a process with UP and DOWN arrows, and thus to use the key k to kill the selected process. I am running Ubuntu 20.04. Some help? Thanks. Pat

**Comment by nicolargo** [comment]
Any relevant message in the log file ?

If you try to kill a process not owned by the Glances user, you should have the following message:

```
2021-02-07 14:38:37,091 -- ERROR -- Can not kill process Plex DLNA Server (psutil.AccessDenied (pid=2198, name='Plex DLNA Server'))
```

Note that Glances only sent a simple kill message to the process.

**Comment by patrcoil** [comment]
Thanks for your answer. No message in the log mode with l key. But, as I mentioned in my previous message, the kill problem is a consequence of the select problem. It is impossible to select any process with UP and DOWN arrows, and thus to kill a process.
